### PR TITLE
Added tests from Call/CallLocal/CreateEvmTxProcessor

### PR DESCRIPTION
**Pull request by rusiboyadjiev**
Signed-off-by: Rusi Boyadjiev <rusiboyadjiev@gmail.com>

### CODE REVIEW

File: hedera-node/src/test/java/com/hedera/services/txns/contract/process/CallEvmTxProcessorTest.java [hunk 1]
```diff
@@ -0,0 +1,172 @@
+package com.hedera.services.txns.contract.process;
```
Comment (Daniel-K-Ivanov) [inline hedera-node/src/test/java/com/hedera/services/txns/contract/process/CallEvmTxProcessorTest.java:1]: Missing license

File: hedera-node/src/test/java/com/hedera/services/txns/contract/process/CallLocalEvmTxProcessorTest.java [hunk 2]
```diff
@@ -0,0 +1,170 @@
+package com.hedera.services.txns.contract.process;
```
Comment (Daniel-K-Ivanov) [inline hedera-node/src/test/java/com/hedera/services/txns/contract/process/CallLocalEvmTxProcessorTest.java:1]: Missing license

File: hedera-node/src/test/java/com/hedera/services/txns/contract/process/CreateEvmTxProcessorTest.java [hunk 3]
```diff
@@ -0,0 +1,166 @@
+package com.hedera.services.txns.contract.process;
```
Comment (Daniel-K-Ivanov) [inline hedera-node/src/test/java/com/hedera/services/txns/contract/process/CreateEvmTxProcessorTest.java:1]: Missing license

**Comment by failfmi** [comment]
Can we add two more tests in Create and Call for the following lines in `EvmTxProcessor`
```java
if (intrinsicGas.toLong() > gasLimit) {
	throw new InvalidTransactionException(gasLimit < dynamicProperties.maxGas() ? INSUFFICIENT_GAS : MAX_GAS_LIMIT_EXCEEDED);
}
```

**Review by failfmi — CHANGES_REQUESTED** [review]
Can we add two more tests in Create and Call for the following lines in `EvmTxProcessor`
```java
if (intrinsicGas.toLong() > gasLimit) {
	throw new InvalidTransactionException(gasLimit < dynamicProperties.maxGas() ? INSUFFICIENT_GAS : MAX_GAS_LIMIT_EXCEEDED);
}
```

**Review by failfmi — APPROVED** [review]

**Pull request merged**